C/src/modal_models/FurutaPendulum/README.md

```diff
@@ -0,0 +1,44 @@
+# Furuta Pendulum
+
+A [Furuta pendulum](https://en.wikipedia.org/wiki/Furuta_pendulum) is a classic control system problem often used to teach feedback control. 
+It consists of a vertical shaft driven by motor, a fixed arm extending out at 90 degrees from the top of the shaft, and a pendulum at the end of the arm.
+The goal is to rotate the shaft to impart enough energy to the pendulum that it swings up, to then catch the pendulum and balance it so that the pendulum remains above the arm.
+Each of these steps requires a different control behavior, which makes a controller a prime candidate for a modal model.
+
+The example here replicates [a solution by Eker et al.](https://ptolemy.berkeley.edu/papers/02/IFAC/IFAC.pdf)
+It uses a simple forward-Euler simulation of an actual pendulum and the control logic provided by Eker et al. to implement a modal controller in Lingua Franca.
```

Comment:
We also have a RK-45 solver due to @YiweiIvy. Should we feature it here?

Reply:
I would say we can leave it as is and maybe create a more advanced variant in the future?

Reply:
But it already works. I expect it is as simple as swapping out the simulated component...

Reply:
I currently do not have the time to adjust the example. I create issue #23 to not further stall this PR and add the solver in a separate PR.